src/bin/create_xion_icaa.rs

```diff
@@ -80,18 +80,19 @@ fn icaa_demo() -> anyhow::Result<()> {
         .account_builder()
         .name("ICAA PL Test")
         // @feedback: this namespace method should note that remote namespaces will not work
+        // @uno-feedback: Which namespace method? You mean we should note that you can't have sub-account from different chain?
```

Comment:
Hm. I believe this was with the thought that we’d have a renote_account method on the builder, and that when you add a namespace there it won’t actually work. However given the remote account builder being separate it should not be an issue. 

Unless I’m failing to remember…

Reply:
Got it, thanks! That brings up a good point not sure why we allow claiming namespace?